.github/workflows/build.yml

```diff
@@ -205,23 +204,29 @@ jobs:
             src/**/Release/*.nupkg
             src/**/Release/*.snupkg
 
+      - name: Install build dependencies
+        if: runner.os == 'Linux' && !matrix.container
+        run: |
+          sudo apt update
+          sudo apt install libcurl4-openssl-dev
+
+      # Run integration tests as soon as possible for non-publishing builds
+      - name: Integration Test
+        if: env.CI_PUBLISHING_BUILD != 'true'
+        uses: getsentry/github-workflows/sentry-cli/integration-test/@v2
+        with:
+          path: integration-test
+
   integration-test:
     needs: build
-    name: Integration test (${{ matrix.container || matrix.os }})
+    name: Integration test (${{ matrix.os }})
     runs-on: ${{ matrix.os }}
-    container: ${{ matrix.container }}
 
     strategy:
       fail-fast: false
       matrix:
         include:
-          - os: ubuntu-22.04 # Pin ubuntu to ensure mono is installed
```

Comment:
What's the motivation behind dropping integration tests on these runners?

Reply:
All runners continue to run integration tests. These were just moved to the .NET build job to avoid waiting for macOS.

![image](https://github.com/user-attachments/assets/e42871c4-eb9e-4868-95f8-1fa8d254c491)